src/services/spotify.js

```diff
@@ -48,3 +49,12 @@ export const search = (query, offset) => {
     .get(endpointUrlSearch, { params })
     .then(res => res.data.tracks.items);
 };
+
+export const topTracks = (offset, time_range = 'medium_term') => {
```

Comment:
We can use constants also for the time range.

Reply:
Yep, had that in mind as well just didn't initially created the constants as we don't have a selector for the time range yet. Added it still